### PR TITLE
Add SatoshiTango

### PR DESCRIPTION
**Pull request by aranciaga**

### CODE REVIEW

**Review by Jawshy — CHANGES_REQUESTED** [review]
@rainbowintheshell, thanks for submitting a pull request. 

Does the site have any documentation about how it has implemented [two-factor authentication](https://github.com/jdavis/twofactorauth/blob/master/CONTRIBUTING.md#a-note-on-definitions)?

**Comment by aranciaga** [comment]
@Jawshy Hi there,

I searched, but I found none.

**Comment by Jawshy** [comment]
Okay. Would you be able to screenshot the process for enabling 2FA and then attach the screenshots below? We can then use that as documentation.

**Comment by aranciaga** [comment]
## Enabling 2-FA in SatoshiTango

Log in, and then go to Settings > Security. Then, click 'Activate'



Scan the QR Code with your mobile using Authy (or similar)



Enter the code, confirm, and you're done!

**Comment by Jawshy** [comment]
Thanks, @rainbowintheshell. Those screenshots show enabling software-based authentication. What is the process for enabling hardware-based authentication?

**Comment by aranciaga** [comment]
@Jawshy My mistake, It's just software-based authentication.

**Review by Jawshy — APPROVED** [review]
:ok_hand: :+1:

File: _data/cryptocurrencies.yml [hunk 1]
```diff
@@ -176,6 +176,13 @@ websites:
       software: Yes
       doc: https://localbitcoins.blogspot.com.au/2013/05/using-two-factor-authentication-on.html
 
+    - name: SatoshiTango
+      url: https://satoshitango.com/
+      img: satoshitango.png
+      tfa: Yes
+      software: Yes
+      doc: https://github.com/2factorauth/twofactorauth/pull/2321#issuecomment-273971890
```
Comment (jamcat22) [inline _data/cryptocurrencies.yml:184]: I'm hesitant to add this as the doc link, considering we have a [notes](https://twofactorauth.org/notes/) section on the website already for sites without pre-made docs. Should we make a new note for this site?

**Review by jamcat22 — CHANGES_REQUESTED** [review]

File: _data/cryptocurrencies.yml (hunk 1 above)
Comment (aranciaga) [reply _data/cryptocurrencies.yml:184]: @jamcat22 I've seen sites with similar doc links, but it's up to you guys. But, creating a note will be more cleaner than this, for sure.

**EDIT**: I noticed that notes doesn't have screenshots. I don't know if it's the idea, but anyway, the pics are uploaded to imgur.

Comment (jamcat22) [reply _data/cryptocurrencies.yml:184]: I'll go ahead and merge this now, and add the PR to our backlog. This way the site appears right away, and there is a doc listed for it, but we can easily return to add a page to the [notes](https://twofactorauth.org/notes/) section later. 😄

**Review by jamcat22 — APPROVED** [review]

**Pull request merged**